core/new-gui/src/app/dashboard/admin/component/execution/admin-execution.component.ts

```diff
@@ -238,21 +236,25 @@ export class AdminExecutionComponent implements OnInit, OnDestroy {
   };
 
   clickToViewHistory(workflowId: number) {
-    const modalRef = this.modalService.open(NgbdModalWorkflowExecutionsComponent, {
-      size: "xl",
-      modalDialogClass: "modal-dialog-centered",
-    });
-
+    let wid!: number;
+    let name!: string;
     for (let i = 0; i < this.workflows.length; i++) {
       const workflow = this.workflows[i];
       if (workflow.wid == workflowId) {
-        let currentWorkflow: Workflow = workflow;
-        modalRef.componentInstance.workflow = currentWorkflow;
-        modalRef.componentInstance.workflowName = currentWorkflow.name;
-
+        wid = workflow.wid;
+        name = workflow.name;
         break;
       }
     }
+
+    this.modalService.create({
+      nzContent: WorkflowExecutionModalComponent,
+      nzComponentParams: { wid: wid },
+      nzTitle: "Execution results of Workflow: " + name,
+      nzFooter: null,
+      nzWidth: "80%",
+      nzCentered: true,
```

Comment:
just wonder if those can be placed in CSS, also why is with only 80%?

Reply:
It's the right way to set the width. CSS is not the correct way to set width for a modal.